packages/core/CHANGELOG.md

```
@@ -12,6 +12,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Keep the package test constraint compatible with the pinned Flutter 3.44.2
   `flutter_test` stack used by release PR checks.
 
+## [4.0.0-dev.8] - 2026-07-31
+
+### Changed
```

markdownlint-cli2 (0.23.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)